app/BUILD.bazel

```diff
@@ -81,7 +132,21 @@ ALTERED_VIEW_MODELS = [
     "src/main/java/org/oppia/app/topic/info/TopicInfoViewModel.kt",
 ]
 
-# Source files for view_model library
+'''
+Source files for the view_models library that DO NOT need to be altered by the view model genrule.
```

Comment:
Here & above: suggest framing the 'place your file here' instructions for view models to crisply specify when a file should go here or above, rather than having the author need to go and read about the genrule in order to understand.

Reply:
Okay